package.json

```diff
@@ -20,12 +20,14 @@
     "expo-status-bar": "~1.2.0",
     "expo-updates": "~0.11.6",
     "moment": "^2.29.1",
+    "native-base": "^3.4.1",
     "prop-types": "^15.8.1",
     "react": "17.0.1",
     "react-native": "0.64.3",
     "react-native-dotenv": "^3.3.1",
     "react-native-get-random-values": "^1.7.2",
     "react-native-qrcode-svg": "^6.1.2",
+    "react-native-svg": "^12.3.0",
```

Comment:
Why is this one being added?

Reply:
Added comment explaining why it´s necessary -- for native base.